### ISSUE TITLE
Dupeguru reporting that exact duplicates that I just made by selecting "Duplicate file" in Finder are not duplicates

### ISSUE THREAD

**Issue by kupietools** [arsenetar/dupeguru#1194]
**Describe the bug**
I have duplicate files. Dupeguru is reporting no duplicates in the folder containing them.

**To Reproduce**
Well, I don't know if it'll reproduce for everybody because I don't know why it doesn't see the duplicates, but:
1. Select a file in Finder
2. Select "Duplicate" from the File menu
3. Open Dupeguru
4. Drag the folder containing the files you just duplicated into Dupeguru's window
5. Select Standard scan by Contents
6. Hit "Scan"
7. Dupeguru reports no duplicates found

**Expected behavior**
Dupeguru reports the just-created duplicate files as duplicate. 

**Screenshots**
<img width="913" alt="Screen Shot 2024-02-11 at 5 18 13 PM" src="https://github.com/arsenetar/dupeguru/assets/26891995/17a779c6-43fe-4860-be46-381026e9ef4c"> 
<img width="843" alt="Screen Shot 2024-02-11 at 5 18 41 PM" src="https://github.com/arsenetar/dupeguru/assets/26891995/bf6d9e7c-eed9-46f9-9972-ca378756392c">

**Desktop (please complete the following information):**
 - OS: MacOS Catalina 10.15.7

**Additional context**
Running `diff` from the terminal on the two files confirms that they are identical, but I just can't get Dupeguru to recognize that they are the same. This is disappointing, I've relied on Dupeguru for many years, and right now have a very big folder heirarchy that was partially duplicated because of a Dropbox sync issue that was not noticed until some of the files had changed in one of the copies, but suddenly my favorite tool for dealing with these situations is no longer working right. 

There's only one line in the debug log from today, from about an hour and a half before this problem occurred: `2024-02-11 15:59:50,353 - WARNING - Couldn't set locale en`

**Comment by kupietools** [comment]
Update: restarting with "debug mode" selected in the preferences seems to have solved the problem for about 30 minutes. (I didn't try restarting without Debug Mode selected so I don't know if that matters.) I'm a little nervous going forward, though, since I now don't know if I can count on this to always find duplicate files. 

About 30 minutes later, it suddenly once again stopped finding exact duplicates made with the Finder's "Duplicate" function. Another restart immediately fixes the problem, and the same search that showed "No duplicates" before quit and restart suddenly shows the duplicates.

Dupeguru appears to now require a quit and restart before each scan for duplicates, otherwise it may not work consistently.

**Comment by kupietools** [comment]
Yeah. There's a big problem here. I'm finding tons of duplicate files in folders that I already ran Dupeguru on.

**Comment by glubsy** [comment]
Have you tried clearing the cache (item in the "File" menu) instead of restarting the application? Does it result in the same behaviour?

**Comment by rkazak** [comment]
I executed the provided steps and was unable to reproduce this issue, I would suggest creating a ( fresh unmodified ) test account and redoing the test there. This would remove anything within your environment as a potential issue, just an idea.

Also my file system is just the local disk.

**Comment by asavageiv** [comment]
I just ran into this too. I tried clearing the cache but it didn't work. Clearing the cache + a restart does seem to work. My bet is Dupeguru is holding onto some file handles for too long.
